src/maliput_osm/osm/conversions.cc

```diff
@@ -34,7 +34,10 @@
 #include <string>
 #include <unordered_set>
```

Comment:
I think we are missing `#include <cmath>`

Reply:
Done.